content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -488,5 +488,7 @@ based on setting `securityContext` within the Pod's `.spec`.
 - [`pod-security.kubernetes.io/exempt`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-exempt)
 - [`pod-security.kubernetes.io/enforce-policy`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-enforce-policy)
 - [`pod-security.kubernetes.io/audit-violations`](/docs/reference/labels-annotations-taints/audit-annotations/#pod-security-kubernetes-io-audit-violations)
+- [`authorization.k8s.io/decision`](/docs/reference/labels-annotations-taints/audit-annotations/#authorization-k8s-io-decision)
+- [`authorization.k8s.io/reason`](/docs/reference/labels-annotations-taints/audit-annotations/#authorization-k8s-io-reason)
```

Comment:
Please add these _before_ the existing annotations, or (better) sort the list alphabetically.

I'll use an example to illustrate the order I'd consider fields for a key:
1. `[['io','k8s','authorization']['decision']]`
1. `[['io','k8s','authorization']['reason']]`
1. `[['io','kubernetes','pod-security']['audit-violations']]`

`k8s` sorts before `kubernetes`.

Reply:
done. PTAL